app.js

```diff
@@ -983,7 +983,10 @@
         values.forEach(function(obj) {
             this.model.set(obj.name, obj.value);
         }.bind(this));
-        // TODO: send model values to server
+        var xhttp = new XMLHttpRequest()
+        xhttp.open("POST", submitURL, true);
+        xhttp.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
+        xhttp.send("data=" + JSON.stringify(this.model));
```

Comment:
XHR is async so this will run off the end of the function without waiting for a server response. Not a big deal perhaps here but might want to put the next two statements in a callback on `onreadystatechange`